### DISCUSSION TITLE
Missing gallery-preloader.gif, see gallery.css

### DISCUSSION THREAD

**Discussion by bpgs** [datenstrom/community#541] [Report a bug] [answered]
In line 575 of gallery.css is a background reference to gallery-preloader.gif  but the file does not exist.

**Comment by schulle4u** [comment]
It looks like this is only needed for backwards compatibility with very old browsers. You can download the file from the [PhotoSwipe repository](https://raw.githubusercontent.com/dimsemenov/PhotoSwipe/master/dist/default-skin/preloader.gif) and upload it as `gallery-preloader.gif` into your extensions folder.

**Comment by markseuffert** [comment]
Can you clarify, if this is something that you found or something that causes a problem for you?

If you're interested in a little bit of technical background, Datenstrom Yellow started a few years ago and at that time there was no support for GIF files. So the GIF file was not included in the gallery and we never heard of an actual problem. As Steffen said, GIF is only needed for very old browsers and such ancient browsers may have died out.

**Comment by markseuffert** [comment]
Ping @bpgs. Have any additional information for us? The more we know the better we can help.

**Comment by markseuffert** [comment, marked as answer]
I could reproduce a 404 error with Windows 10 and updated the extension as suggested, please try again.
